### PR TITLE
fix: addresses issue causing mismanaged disabled state

### PR DESCRIPTION
**Pull request by nicholasrice**
# Description
Disabled state for text-field and text-area were not being applied correctly.

## Motivation & context
TextField clobbered the disabled attr defined by FromAssociated with a different config, which clobbered the converter and resulted in un-expected disabled values.

Also updates several other components which could have similar issues due to re-defining the attr on an extended property.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-components/src/text-area/text-area.ts [hunk 1]
```diff
@@ -56,25 +56,17 @@ export class TextArea extends FormAssociated<HTMLTextAreaElement> {
...
+    @attr({ mode: "boolean" })
     public autofocus: boolean;
     private autofocusChanged(): void {
         if (this.proxy instanceof HTMLElement) {
             this.proxy.autofocus = this.autofocus;
         }
     }
 
-    @attr
+    @attr({ converter: nullableNumberConverter, mode: "fromView" })
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/text-area/text-area.ts:55]: are these not needed on textfield as well?

File: packages/web-components/fast-components/src/text-field/text-field.ts [hunk 2]
```diff
@@ -108,8 +107,6 @@ export class TextField extends FormAssociated<HTMLInputElement> {
         }
     }
 
-    @attr
-    public value: string;
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/text-field/text-field.ts:112]: is the duplication of these from the inherited abstract class causing the issue?

Comment (nicholasrice) [reply packages/web-components/fast-components/src/text-field/text-field.ts:112]: it was caused by defining a different converter for the inherited property, yes - multiple `@attr` on the property is a no-no.

I've cleaned them all up except for slider which converts the `value` prop to a number. We'll want to address that but I think it is a larger change we should isloate.

File: packages/web-components/fast-components/src/text-area/text-area.ts (hunk 1 above)
Comment (nicholasrice) [reply packages/web-components/fast-components/src/text-area/text-area.ts:55]: fixed.

File: packages/web-components/fast-components/src/text-field/text-field.ts (hunk 2 above)
Comment (chrisdholt) [reply packages/web-components/fast-components/src/text-field/text-field.ts:112]: gotcha - assuming the issue would persist if the `@attr` had a different default value here which may be something that needs to be addressed as well?

Comment (nicholasrice) [reply packages/web-components/fast-components/src/text-field/text-field.ts:112]: nm pulling it off slider seems to work for the examples so pulled it off. There might be some bugs but I think we can harden it in a second pass.

File: packages/web-components/fast-components/src/slider/slider.ts [hunk 3]
```diff
@@ -50,7 +50,7 @@ export class Slider extends FormAssociated<HTMLInputElement>
     /**
      * The element's value to be included in form submission changed.
      */
-    @attr({ converter: nullableNumberConverter })
+    // @attr({ converter: nullableNumberConverter })
```
Comment (janechu) [inline packages/web-components/fast-components/src/slider/slider.ts:53]: should we just remove this line?

Comment (chrisdholt) [reply packages/web-components/fast-components/src/slider/slider.ts:53]: looks like an artifact from removing from slider :)

Comment (nicholasrice) [inline packages/web-components/fast-components/src/slider/slider.ts:53]:
```suggestion
```

**Pull request merged**